app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html

```diff
@@ -120,7 +120,7 @@
                         </li>
                         <!--<li>
                             <a
-                                ui-state="'appWithoutMenus.insights'"
+                                ui-state="'appReact.insights'"
```

Comment:
This is the wrong state after you renamed it.
```suggestion
                                ui-state="'app.ministries.ministry.insights'"
```